controllers/gitops_reconciler.go

```diff
@@ -21,15 +21,34 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 )
 
+var (
+	secretName              = "argocd-default-cluster-config"
+	clusterConfigSecretData = map[string]string{}
+)
+
+const (
+	ARGOCD_NAMESPACE_NAME             = "argocd"
+	GITOPS_SUBSCRIPTION_NAME          = "openshift-gitops-operator"
+	GITOPS_SUBSCRIPTION_PACKAGE_NAME  = "openshift-gitops-operator"
+	GITOPS_OPERATOR_NAMESPACE_NAME    = "openshift-operators"
+	GITOPS_DEPLOYMENT_STATUS_NAME     = "gitops-operator"
+	GITOPS_ROLE_NAME                  = "argocd-manager"
+	GITOPS_ROLEBINDING_NAME           = "argocd-manager"
+	GITOPS_ROLE_KIND_NAME             = "Role"
```

Comment:
this is how we should name?

Reply:
@rasheedamir  i have updated  constant name.